modules/react/tokens/lib/type/index.ts

```diff
@@ -0,0 +1,199 @@
+import {fonts, CanvasFonts} from './fonts';
+import {fontSizes, CanvasFontSizes} from './fontSizes';
+import {fontWeights, CanvasFontWeights} from './fontWeights';
+import {levels, CanvasTypeHierarchy} from './levels';
+import {variants, CanvasTypeVariants} from './variants';
+
+/** ### Canvas Type Properties
+ * [View Storybook Docs](https://workday.github.io/canvas-kit/)
+ *
+ * ---
+ * The type properties provide precise control and are only intended to be used when the type hierarchy tokens are not well-suited for the task.
+ *
+ * The properties are organized in three parts:
+ *
+ * - `fonts` (font families)
+ * - `fontSizes`
+ * - `fontWeights`
+ *
+ * You can find more detailed information by inspecting individual properties.
+ */
+const properties: CanvasTypeProperties = {
+  fonts,
```

Comment:
I think this is a good call out. I want to avoid the use of `sizes` in properties as we're also using that term for "small, medium, and large steps within each level of the type hierarchy." That was probably the hardest mental switch I had to make to feel comfortable with the new hierarchy. 

But you're correct that this is inconsistent. Two properties follow the CSS name convention, and one, `fonts`, does not. I would prefer to use `fontFamilies` instead.